src/source-napatech.c

```diff
@@ -770,17 +789,29 @@ TmEcode NapatechPacketLoop(ThreadVars *tv, void *data, void *slot)
     uint64_t pkt_ts;
     NtNetBuf_t packet_buffer;
     NapatechThreadVars *ntv = (NapatechThreadVars *) data;
+    NtFlowStream_t flow_stream[MAX_ADAPTERS];
```

Comment:
Suggest moving this within the guard `#ifdef NAPATECH_ENABLE_BYPASS`.
Suggest using `= {0}` to zero-fill.